Makefile

```diff
@@ -21,6 +21,9 @@ export GOPROXY=https://proxy.golang.org
 # Disable CGO so that we always generate static binaries:
 export CGO_ENABLED=0
 
+# Allow overriding: `make lint container_runner=podman`.
+container_runner:=docker
```

Comment:
can we use podman instead?

Reply:
It gave some problem, probably too old in ci-ext Jenkins. Let me try again...